spec.html

```diff
@@ -34164,77 +34164,54 @@ <h1>Array.prototype.some ( _callbackfn_ [ , _thisArg_ ] )</h1>
           </li>
-        </ul>
-        <p>The sort order is also implementation-defined if any of the following conditions are true:</p>
-        <ul>
           <li>
-            If _obj_ is an exotic object (including Proxy exotic objects) whose behaviour for [[Get]], [[Set]], [[Delete]], and [[GetOwnProperty]] is not the ordinary object implementation of these internal methods.
-          </li>
-          <li>
-            If any index property of _obj_ whose name is a nonnegative integer less than _len_ is an accessor property or is a data property whose [[Writable]] attribute is *false*.
-          </li>
-          <li>
-            If _comparefn_ is *undefined* and the application of ToString to any value passed as an argument to SortCompare modifies _obj_ or any object on _obj_'s prototype chain.
+            If _comparefn_ is *undefined* and SortCompare does not act as a consistent comparison function.
           </li>
           <li>
             If _comparefn_ is *undefined* and all applications of ToString, to any specific value passed as an argument to SortCompare, do not produce the same result.
           </li>
         </ul>
-        <p>The following steps are taken:</p>
-        <emu-alg>
-          1. Perform an implementation-dependent sequence of calls to the Get, <emu-xref href="#sec-set-o-p-v-throw">Set</emu-xref>, DeletePropertyOrThrow, and HasOwnProperty abstract operation with _obj_ as the first argument, and to SortCompare (described below), such that:
-            * The property key argument for each call to Get, <emu-xref href="#sec-set-o-p-v-throw">Set</emu-xref>, HasOwnProperty, or DeletePropertyOrThrow is the string representation of a nonnegative integer less than _len_.
-            * The `Throw` argument for every call to <emu-xref href="#sec-set-o-p-v-throw">Set</emu-xref> is *true*.
-            * The arguments for calls to SortCompare are values returned by a previous call to the Get abstract operation, unless the properties accessed by those previous calls did not exist according to HasOwnProperty. If both prospective arguments to SortCompare correspond to non-existent properties, use *+0* instead of calling SortCompare. If only the first prospective argument is non-existent use +1. If only the second prospective argument is non-existent use -1.
-            * If _obj_ is not sparse then DeletePropertyOrThrow must not be called.
-            * If an abrupt completion is returned from any of these operations, it is immediately returned as the value of this function.
-          1. Return _obj_.
-        </emu-alg>
-        <p>Unless the sort order is specified above to be implementation-defined, the returned object must have the following two characteristics:</p>
+        <p>Unless the sort order is specified above to be implementation-defined, _items_ must have the following two characteristics after executing step 8 of the algorithm above:</p>
```

Comment:
Editorial nit: rather than giving the explicit number of the step, you can label the step itself:

```
1. [id="step-array-sort"] Sort _items_ using an implementation-dependent sequence of calls to SortCompare.
```
and then reference it:
```
[...] after executing step <emu-xref href="#step-array-sort"></emu-xref> of the algorithm above [...]
```

This ensures the number doesn't get stale.

Reply:
Done